templates/mail/issue/assigned.tmpl

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+	<title>{{.Subject}}</title>
+</head>
+
+<body>
+	<p>@{{.Doer.Name}} assigned you to the {{if eq .Issue.IsPull true}}pull request{{else}}issue{{end}}  <a href="{{.Link}}">#{{.Issue.Index}}</a>.</p>
```

Comment:
I know it's in the subject, but could it be better to include the name of the repository here as well?

Reply:
Added.